samples/SkiaMultiplatformSample/build.gradle.kts

```diff
@@ -110,12 +115,28 @@ kotlin {
             }
         }
 
-        val jsMain by getting {
+        val jsWasmMain by creating {
             dependsOn(commonMain)
             resources.setSrcDirs(resources.srcDirs)
             resources.srcDirs(unzipTask.map { it.destinationDir })
         }
 
+        val jsMain by getting {
+            dependsOn(jsWasmMain)
+            dependencies {
+                implementation("org.jetbrains.skiko:skiko-js:$version")
```

Comment:
if commonMain has `implementation("org.jetbrains.skiko:skiko:$version")` than jsMain and wasmMain probably don't need to add this dependency explicitly? 
Or it doesn't work like that for k/wasm?

Reply:
Yes, we can remove whole dependencies block as well. Will do.